pkgs/development/libraries/bearssl/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  version = "0.6";
+  sha256 = "057zhgy9w4y8z2996r0pq5k2k39lpvmmvz4df8db8qa9f6hvn1b7";
+
+in
+stdenv.mkDerivation {
+  pname = "bearssl";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://www.bearssl.org/bearssl-${version}.tar.gz";
+    inherit sha256;
+  };
+
+  outputs = [ "bin" "lib" "dev" "out" ];
```

Comment:
Wouldn't `lib` be a more appropriate default output?

Reply:
`bin` is always expected to come first, the code that needs lib usually knows to default to it.

Reply:
Fair enough :)